index.html

```diff
@@ -2141,6 +2173,268 @@ <h2>
         This section is a work in progress as this document evolves.
       </p>
     </section>
+    <section id="user-agent-automation" data-cite="webdriver">
+      <h2>
+        User Agent Automation
+      </h2>
+      <p>
+        For the purposes of user agent automation and web application testing,
+        this document defines a number of [[WebDriver]] [=extension commands=].
+      </p>
+      <section id="webdriver-extension-capability">
+        <h3>
+          Digital Credentials WebDriver Extension Capability
+        </h3>
+        <p>
+          In order to advertise the availability of [=extension commands=]
+          defined below, a new extension capability is defined.
+        </p>
+        <table class="data">
+          <thead>
+            <tr>
+              <th>
+                Capability
+              </th>
+              <th>
+                Key
+              </th>
+              <th>
+                Value Type
+              </th>
+              <th>
+                Description
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                Virtual Wallet Support
+              </td>
+              <td>
+                <code>digital-credentials:virtualWallet</code>
+              </td>
+              <td>
+                [=boolean=]
+              </td>
+              <td>
+                Indicates whether the endpoint node supports all Virtual Wallet
+                commands.
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <p>
+          When <a data-cite="webdriver#dfn-validate-capabilities">validating
```

Comment:
nit: avoid using data-cite... if something hasn't been exported from Web Driver, we should send a PR to web driver to export the definition we need.